CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.1
```

Comment:
i don't think this version has been pushed to rubygems :( https://rubygems.org/gems/rack-timeout